<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>804</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>817</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>8514oem</family>
    <pointsize>8</pointsize>
    <weight>75</weight>
    <italic>false</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Game Window</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(spread:pad, x1:0.483, y1:0, x2:0.539773, y2:1, stop:0 rgba(255, 180, 104, 255), stop:1 rgba(255, 131, 131, 255))</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="mainWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLineEdit" name="UserInputBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>760</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Poor Richard&quot;;
background-color: qlineargradient(spread:pad, x1:0.505682, y1:1, x2:0.494318, y2:0.011, stop:0 rgba(255, 231, 131, 255));</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InputButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>760</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Poor Richard&quot;;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:1, stop:1 rgba(255, 162, 104, 255));</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="locationBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Poor Richard&quot;;
border: 1px solid grey</string>
    </property>
    <property name="text">
     <string>Location:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="heroNameBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poor Richard</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Poor Richard&quot;;
border: 1px solid grey

</string>
    </property>
    <property name="text">
     <string>Hero:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="loginPanel" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>500</y>
      <width>281</width>
      <height>201</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>281</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Poor Richard&quot;;
border: 1px solid grey</string>
    </property>
    <widget class="QLineEdit" name="UsernameInput">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PasswordInput">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="LoginButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Poor Richard&quot;;
background-color: qlineargradient(spread:pad, x1:0.483, y1:0, x2:0.539773, y2:1, stop:0 rgba(255, 166, 111, 255), stop:0.676136 rgba(255, 231, 131, 255));
</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="NewAccountButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poor Richard</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Poor Richard&quot;;
background-color: qlineargradient(spread:pad, x1:0.483, y1:0, x2:0.539773, y2:1, stop:0 rgba(255, 166, 111, 255), stop:0.676136 rgba(255, 231, 131, 255));
</string>
     </property>
     <property name="text">
      <string>Create Account</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>571</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Poor Richard&quot;;
background-color: qlineargradient(spread:pad, x1:0.483, y1:0, x2:0.539773, y2:1, stop:0 rgba(255, 166, 111, 255), stop:0.676136 rgba(255, 231, 131, 255));
</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <zorder>UserInputBox</zorder>
   <zorder>InputButton</zorder>
   <zorder>locationBox</zorder>
   <zorder>heroNameBox</zorder>
   <zorder>textDisplay</zorder>
   <zorder>loginPanel</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
